emission/tests/storageTests/TestTokenQueries.py

```diff
@@ -0,0 +1,91 @@
+import unittest
+import logging
+import uuid
+import json
+
+import os
+
+import emission.core.get_database as edb
+
+import emission.storage.decorations.token_queries as esdt
+import emission.net.auth.token_list as enat
+
+import bin.auth.insert_tokens as bai
+
+class TestTokenQueries(unittest.TestCase):
+    def setUp(self):
+        self.testUserId = uuid.uuid3(uuid.NAMESPACE_URL, "mailto:test@test.me")
+        edb.get_token_db().delete_many({})
```

Comment:
This should really be in `teardown`

```suggestion

def tearDown(self):
    # delete all the tokens added here so that if we inadvertently run the tests against a real database, we don't
    # lose real data. There doesn't seem to be a pre-defined list of tokens - there is one file but there are also tests
    # that add `1`, `2` etc. So let's delete all single character tokens
    # so something like https://stackoverflow.com/questions/29577713/string-field-value-length-in-mongodb
    pass
```